platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxMap.java

```diff
@@ -274,6 +281,31 @@ public boolean getPrefetchesTiles() {
     return nativeMapView.getPrefetchTiles();
```

Comment:
`MapboxMap#getPrefetchesTiles` could also be deprecated.

Reply:
@LukasPaczos I thought this feature will be still working as expected and won't interfere with the new functions, but maybe it's better to deprecate it as well to be consistent.